private/projects/aws-arch.ts

```diff
@@ -0,0 +1,98 @@
+/*********************************************************************************************************************
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ******************************************************************************************************************** */
+import { Project, TaskStep } from "projen";
+import { Stability } from "projen/lib/cdk";
+import { PDKProject } from "../pdk-project";
+
+/**
+ * Contains configuration for the BarneoProject.
+ */
+export class AwsArchProject extends PDKProject {
+  constructor(parent: Project) {
+    super({
+      parent,
+      author: "AWS APJ COPE",
+      authorAddress: "apj-cope@amazon.com",
+      defaultReleaseBranch: "mainline",
+      name: "aws-arch",
+      keywords: ["aws", "pdk", "jsii", "projen", "icons", "assets", "cfnspec"],
+      repositoryUrl: "https://github.com/aws/aws-prototyping-sdk",
+      devDeps: [
+        "@aws-cdk/cfnspec",
+        "@types/node-fetch@2",
+        "@types/sharp",
+        "@types/unzipper",
+        "@types/xml-flow",
+        "node-fetch@2",
+        "projen",
+        "sharp",
+        "tree-cli",
+        "unzipper",
+        "xml-flow",
+      ],
+      peerDeps: ["projen"],
+      bundledDeps: ["lodash", "traverse", "fs-extra"],
```

Comment:
Do these need to be bundled? Are we exposing lodash, traverse or fs-extra in any interfaces?

Reply:
Good catch! No they are not, was left over from refactor of the other projects inflight. `traverse` is not even used in this package and the others are dev deps only.